ibmcloud/image/Makefile

```diff
@@ -17,25 +17,25 @@ IMAGE_SUFFIX := .qcow2
 
 build: $(IMAGE_FILE)
 
-push: $(IMAGE_FILE)
+push:
```

Comment:
With the changes to the makefile why does the push now no longer do the build?

Reply:
if we run `make build` and then `make push` with the old Makefile, the pod-vm-image will be build twice, so that remove the dependency here.

Reply:
I think with the removal of the se- prefix it will no longer do that if there is an existing file that matches the IMAGE_NAME.

Reply:
yeah, verified, rollback the change for Makefile.